Src/ILGPU/Runtime/Accelerator.cs

```diff
@@ -87,6 +87,30 @@ public static AcceleratorType CurrentType
             get => AcceleratorType.CPU;
         }
 
+        /// <summary>
+        /// Returns the version number of the inline language for the current accelerator.
+        /// </summary>
+        /// <remarks>
+        /// Note that this static property is also accessible within kernels.
+        /// </remarks>
+        internal static int LanguageVersion
+        {
+            [LanguageIntrinsic(LanguageIntrinsicKind.Version)]
+            get => 0;
+        }
+
+        /// <summary>
+        /// Converts the major/minor version into a single language version number.
+        /// </summary>
+        /// <param name="major">The major version.</param>
+        /// <param name="minor">The minor version.</param>
+        /// <returns>The language version number</returns>
+        /// <remarks>
+        /// The format is arbitrary, and assumes there are at most 99 minor versions.
+        /// </remarks>
+        internal static int MakeLanguageVersion(int major, int minor) =>
```

Comment:
A got it... do you think an "arbitrary" version structure for ILGPU-specific inline language elements make sense (see comment about the `enumeration` above).

Reply:
I was originally using `int MajorVersion` and `int MinorVersion`. This allowed easy matching in `IsPTXVersionAtLeast(major, minor)`. Later, I changed to a single `int LanguageVersion` so that the accelerator specialisation did not have to do two passes.

What would this enumeration look like? And how would we map `IsPTXVersionAtLeast(major, minor)` to this enumeration?

Reply:
Sounds good 👍 Maybe we can encapsulate the `int` in a parent structure that can be extended in the future without patching all uses of `int` 🤓

Reply:
Changed to an opaque enum (`AcceleratorLanguageVersion`) to make future refactoring easier.